rfcs/text/0040-volume-device.md

```diff
@@ -49,6 +44,133 @@ Stage X: Provide a brief explanation of why the proposal is being marked as aban
 <!--
 Stage 1: Describe at a high level how this change affects fields. Include new or updated yml field definitions for all of the essential fields in this draft. While not exhaustive, the fields documented here should be comprehensive enough to deeply evaluate the technical considerations of this change. The goal here is to validate the technical details for all essential fields and to provide a basis for adding experimental field definitions to the schema. Use GitHub code blocks with yml syntax formatting, and add them to the corresponding RFC folder.
 -->
+Details of the proposed fields:
+
+```
+---
+- name: volume
+  title: Volume
+  group: 2
+  short: Fields relevant to storage volumes.
+  description: >
+    Fields that describe the storage volumes.
+  type: group
+  fields:          
+    - name: mount_name
+      level: extended
+      type: keyword
+      description: >
+        Mount name of the volume device.
+        The field is relevant to Posix only.
+ 
+    - name: device_name
+      level: extended
+      type: keyword
+      description: >
+        Full path of the device.
+        The field is relevant to Posix only.
+
+    - name: dos_name
+      level: extended
+      type: keyword
+      short: DOS name of the device.
+      description: >
+        DOS name of the device.
+        DOS device name is in the format of driver letters such as C:, D:,...
```

Comment:
Would it make sense for `volume.dos_name` to follow the convention established by [`file.drive_letter`](https://www.elastic.co/guide/en/ecs/current/ecs-file.html#field-file-drive-letter) and suggest (from the `file.drive_letter` description's guidance) that `The value should be uppercase, and not include the colon.`? Or is the literal device name (including the colon) significant for this use case?

Reply:
changed description per suggestion: https://github.com/elastic/ecs/pull/2229/commits/ddfebdee54fc6df5f7b51846fdf43cbfd6464366
We have being using uppercase letter followed by colon for quite some years.